### PR TITLE
Update curator docs to 4.0

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1685052
https://bugzilla.redhat.com/show_bug.cgi?id=1685044

### CODE REVIEW

**Comment by mburke5678** [comment]
@QiaolingTang @xltian If one of both of you can please review my changes to bring the docs in line with 4.0 I would greatly appreciate it.

File: modules/efk-logging-curator-configuration.adoc [hunk 1]
```diff
@@ -21,10 +21,10 @@ Set cluster logging to the unmanaged state.
 
 .Procedure
 
-. To configure Curator, edit the `logging-curator` configuration map:
+. To configure Curator, edit the `curator` configuration map in the `cluster-logging` project:
```
Comment (QiaolingTang) [inline modules/efk-logging-curator-configuration.adoc]: Maybe `openshift-logging` project?

Comment (mburke5678) [reply modules/efk-logging-curator-configuration.adoc]: @QiaolingTang Fixed. Thank you.

**Comment by mburke5678** [comment]
@QiaolingTang Can I ask you to look at this commit where I reorganized the section we spoke about in Slack?
https://github.com/openshift/openshift-docs/pull/14027/commits/efb4a8ba8114d4730b46034ca758e76f25a0e3bb

File: modules/efk-logging-curator-cron.adoc [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+* Alternatively, if you are using the *actions file*:
++
+.. Copy the Curator configuration file, the {produxt-title} custom configuration file, and the
+the {produxt-title} *action* file from the Curator configuration map and create separate files for each:
++
+----
+$ oc extract configmap/logging-curator --keys=curator5.yaml,actions.yaml --to=/my/config
+----
++
+.. Edit the *_/my/config/curator5.yaml_* and *_/my/config/config.yaml_* files.
```
Comment (QiaolingTang) [inline modules/efk-logging-curator-cron.adoc]: and /my/config/config.yaml files. ----> and /my/config/actions.yaml files.

File: modules/efk-logging-curator-cron.adoc [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
++
+.. Delete the existing Curator config map and add the edited YAML files to a new Curator config map.
++
+----
+$ oc delete configmap curator ; sleep 1
+$ oc create configmap curator \
+    --from-file=curator5.yaml=/my/config/curator5.yaml \
+    --from-file=config.yaml=/my/config/config.yaml \
```
Comment (QiaolingTang) [inline modules/efk-logging-curator-cron.adoc]: should be : --from-file=actions.yaml=/my/config/actions.yaml \

Comment (mburke5678) [reply modules/efk-logging-curator-cron.adoc]: @QiaolingTang Thank you. Time to stop for the day. Making errors from being tired!

**Pull request merged**